docs: clarify Go environment setup in local development guide (fixes #954) (#1021)
gittuf/gittuf: docs/get-started.md

> `ezwinports.make` package: Simply type `winget install ezwinports.make`
> in PowerShell.
> You can also install it from the [GNU website] or the [chocolatey] package manager.

> >
> > [!IMPORTANT]
> > Before building from source, ensure that your Go environment is correctly set up.
> > You can do this by following the official [Go installation instructions](https://go.dev/doc/install).
> >
> > Specifically, make sure your `GOPATH` and `PATH` are configured correctly
> 
> 

Reviewer: I think this would be sufficient. The link to the installation instructions should be added to the list at the bottom of the file. Please also make sure that the changes are wrapped to 80 cols.

```suggestion
Before building from source, please ensure that your Go environment is correctly set up.
You can do this by following the official [Go installation instructions].
If you encounter any issues when building, make sure your `GOPATH` and `PATH` are configured correctly.

```

Author: I've implemented the changes
